clp_ffi_py/__init__.py

```diff
@@ -1,2 +1,3 @@
 from clp_ffi_py.ir import *  # noqa
 from clp_ffi_py.wildcard_query import *  # noqa
+from clp_ffi_py.query_builder import *  # noqa
```

Comment:
Alphabetize

Reply:
sorry I forgot ruff is disabled for this file